MAINTAINERS

```diff
@@ -383,6 +385,62 @@ made through a pull request.
 				"duglin"
 			]
 
+		[Org.Subsystem.daemon]
+
+			people = [
+				"cpuguy83",
+				"crosbymichael",
+				"shykes",
+				"tibor",
+				"unclejack",
+				"vieux",
+				"vishh",
+			]
+
+		[Org.Subsystem.engine]
+
+			people = [
+				"shykes"
+			]
+
+		[Org.Subsystem.graph]
+
+			people = [
+				"crosbymichael",
+				"shykes",
+				"tibor",
+				"unclejack",
+				"vieux",
+			]
+
+		[Org.Subsystem.integration]
+
+			people = [
+				"tibor",
+				"unclejack"
+			]
+
+		[Org.Subsystem.integration-cli]
+
+			people = [
+				"unclejack"
+			]
+
+		[Org.Subsystem.project]
+
+			people = [
+				# NOTE: jpetazzo is maintainer for "dind"
+				"jpetazzo",
```

Comment:
I wasn't sure how this information should be included in  the document and decided to just add a comment. Possibly this should become a sub-sub-system? `Org.Subsystem.project.dind` or similar?